samples/unity_v230_ops_48.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: storage.dell.com/v1
```

Comment:
This release we support OCP 4.9 and 4.10. so we should not be having a sample for OCP 4.8